wagtail_ab_testing/templates/wagtail_ab_testing/add_form.html

```diff
@@ -34,6 +34,8 @@
     <script>
         {% url 'wagtailadmin_home' as wagtailadmin_root_url %}
         wagtailConfig.ADMIN_ROOT_URL = '{{ wagtailadmin_root_url|escapejs }}';
+        {% url 'wagtailadmin_choose_page' as wagtailadmin_choose_page_url %}
+        window.AB_TESTING_CHOOSE_PAGE_URL = '{{ wagtailadmin_choose_page_url|escapejs }}';
```

Comment:
I'd rather see us change this to something that is inherently compatible with strict [content security policies](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP).

@mgax could you change this to use [`json_script`](https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#std-templatefilter-json_script) instead?

Reply:
Do you think that the patten on the above two lines is okay, i.e. setting the value on `wagtailConfig`? If not, I'd better change that one too.

https://github.com/wagtail-nest/wagtail-ab-testing/blob/cbeff6f982bdb94bedd001182b9b174c4546b867/wagtail_ab_testing/templates/wagtail_ab_testing/add_form.html#L35-L36

Reply:
Yes, changing that too would make sense now that I look at it more closely!

Checking its usage, it appears to only be used to construct a URL to fetch page information: https://github.com/wagtail-nest/wagtail-ab-testing/blob/7e5bb952d6ed86fb2d63166fd08cb81d41668613/wagtail_ab_testing/static_src/components/GoalSelector/index.tsx#L40

`ADMIN_ROOT_URL` is internal to us, there is no reference inside the (current) Wagtail codebase. No need to put this in `wagtailConfig` I feel.

Reply:
Should be fixed now.